arrow/locales.py

```diff
@@ -285,6 +295,18 @@ def _format_relative(
 
         return direction.format(humanized)
 
+    def _format_brief(
+        self,
+        humanized: str,
+    ) -> str:
+
+        parts = humanized.split()
+
+        if parts[0].isnumeric():
+            return parts[0] + parts[1][0]
+
+        return parts[1][0]
+
```

Comment:
Could there be a case for creating a `timeframes_brief` dict for each locale that supports this feature?

I'm not sure how locales outside of English deal with shortening.